src/Xamarin.Android.Build.Tasks/Tasks/Crunch.cs

```diff
@@ -9,86 +9,158 @@
 using System.Text.RegularExpressions;
 using Xamarin.Android.Build.Utilities;
 using Xamarin.Android.Tools.Aidl;
+using ThreadingTasks = System.Threading.Tasks;
 
 namespace Xamarin.Android.Tasks
 {
-	public class Crunch : ToolTask
+	public class Crunch : AsyncTask
 	{
+		private const string error_regex_string = @"(?<file>.*)\s*:\s*(?<line>\d*)\s*:\s*error:\s*(?<error>.+)";
```

Comment:
We should follow normal naming conventions; this should be PascalCased `ErrorRegexString`.